jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -234,18 +228,6 @@ public void addCookie(Cookie cookie)
         String comment = cookie.getComment();
         boolean httpOnly = cookie.isHttpOnly();
 
-        if (comment != null)
```

Comment:
Hmmm by my own reasoning... we never needed to check the comment here because Cookie now has support for it.... so we probably need to deprecate support for that an remove in a future release.

However it doesn't have support for sameSite, so now you should call the statics here to get the sameSite and strip the comment.

So in summary, it is here that I think  you should be calling the new static methods.